asdf/tests/test_schema.py

```diff
@@ -775,6 +775,64 @@ def test_custom_validation_with_definitions_bad(tmpdir):
             pass
 
 
+def test_custom_validation_with_external_ref_good(tmpdir):
+    custom_schema_path = helpers.get_test_data_path('custom_schema_external_ref.yaml')
+    asdf_file = os.path.join(str(tmpdir), 'out.asdf')
```

Comment:
```
asdf_file = str(tmpdir.join("out.asdf"))
```
is the more compact way to do this, using the `join()` method on the path object supplied by the `tmpdir` fixture.

Reply:
Nice, I'll update this